features/workspace/products/product/compliance-information/ManageLanguagesDialog.tsx

```
@@ -0,0 +1,384 @@
+    });
+  };
+
+  const handleSelectAllVisible = () => {
+    setSelectedCodes((current) => {
+      const currentSet = new Set(current);
+
+      if (allVisibleSelected) {
+        return current.filter((code) => !visibleCodes.includes(code));
+      }
+
+      visibleCodes.forEach((code) => currentSet.add(code));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)